src/Android/Xamarin.Android/Samples/Network Analysis/ClosestFacility/ClosestFacility.cs

```diff
@@ -0,0 +1,207 @@
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Android.App;
+using Android.OS;
+using Android.Widget;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.Tasks.NetworkAnalysis;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.UI.Controls;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ArcGISRuntime.Samples.ClosestFacility
+{
+    [Activity(Label = "ClosestFacility")]
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Closest facility (interactive)",
+        "Network Analysis",
+        "Demonstrates how to solve a Closest Facility Task to find the closest route between a facility (hospital) and a incident (black cross). Tap to find the route to the nearest hospital.",
+        "")]
+    public partial class ClosestFacility : Activity
+    {
+        // Map view that will be used
+        private MapView _myMapView = new MapView();
+
+        // Holds locations of hospitals around San Diego
+        private List<Facility> _facilities;
+
+        // Graphics overlays for facilities and incidents
+        private GraphicsOverlay _facilityGraphicsOverlay;
+
+        // Symbol for facilities
+        private PictureMarkerSymbol _facilitySymbol;
+
+        private GraphicsOverlay _incidentGraphicsOverlay;
+
+        // Black cross where user clicked
+        private MapPoint _incidentPoint;
+
+        // Symbol for the incident
+        private SimpleMarkerSymbol _incidentSymbol;
+
+        // Used to display route between incident and facility to mapview
+        private SimpleLineSymbol _routeSymbol;
+
+        // Solves task to find closest route between an incident and a facility
+        private ClosestFacilityTask _task;
+
+        protected override void OnCreate(Bundle bundle)
+        {
+            base.OnCreate(bundle);
+
+            Title = "Closest Facility";
+
+            // Initialize the app
+            Initialize();
+
+            // Create the UI
+            CreateLayout();
+        }
+
+        private void CreateLayout()
+        {
+            // Create a new layout for the entire page
+            var layout = new LinearLayout(this) { Orientation = Orientation.Vertical };
+
+            // Add the map view to the layout
+            layout.AddView(_myMapView);
+
+            // Show the layout in the app
+            SetContentView(layout);
+        }
+
+        private void Initialize()
+        {
+            // Hook up the DrawStatusChanged event.
+            _myMapView.DrawStatusChanged += OnDrawStatusChanged;
+
+            // Load the basemap.
+            Map map = new Map(Basemap.CreateLightGrayCanvasVector());
```

Comment:
We've followed this pattern in the past, but I don't think it's helpful. I've been changing these to construct the map and set the MapView.Map property on one line.